style.css

```diff
@@ -0,0 +1,204 @@
+/* font-family: 'Work Sans', sans-serif;*/
+*{
+    box-sizing: border-box;
+}
+:root{
+    --clr-primary: #f70058;
+    --clr-dark: #0c0e36;
+    --clr-light: #fff;
+    --fw-bold: 400;
+    --fw-bold: 700;
+    --fw-black: 900;
+    --grid: 1fr;
+    --grid: minmax(1em,10%)
+            minmax(10rem,22ch)
+            minmax(10rem,23ch)
+            minmax(1em,3fr)
+            max-content;
+}
+body{
+    margin: 0%;
+    font-family: 'Work Sans' , sans-serif;
+    font-size: 1.125rem;
+
+    color: var(--clr-dark);
+    line-height: 1.6;
+}
+header{
+    background-color: var(--clr-light);
+    text-align: center;
+}
+
+.nav_list{
+    display: flex;
+    justify-content: space-between;
+
+    padding: 1em;
+    margin: 0%;
+    /*margin-top: 1em;*/
```

Comment:
remove comments

Reply:
ok

Reply:
not only this comment but remove all from PR...